app/components/collections/edit/contact_emails_component.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Collections
+  module Edit
+    # Component for editing collection contact emails
+    class ContactEmailsComponent < ApplicationComponent
+      def initialize(form:)
+        @form = form
+        super()
+      end
+
+      attr_reader :form
+    end
+  end
+end
```

Comment:
We now need work- and collection-specific contact email components because they have different tooltip values.